### DISCUSSION TITLE
Is it expected Tooltip to have dependency on @internationalized/date?

### DISCUSSION THREAD

**Discussion by ohazda** [adobe/react-spectrum#3663] [Q&A] [answered]
Hey, we have been using Tooltips and realized that the size of the package grew by around 20% and now @internationalized/date is being bundled together. 

I was using this as an analyzer: https://bundlephobia.com/package/@react-spectrum/tooltip@3.1.6.

Just wanted to double-check if it's expected because it doesn't look like this package depends on @internationalized/date.

Thank you!

**Comment by snowystinger** [comment]
~I ran yarn why, I do not see any reason for tooltip having it in its bundle. We'll need to dig some more.~

   - Specified in "dependencies"
   - Hoisted from "_project_#@internationalized#date"
   - Hoisted from "_project_#@react-aria#calendar#@internationalized#date"
   - Hoisted from "_project_#@react-aria#datepicker#@internationalized#date"
   - Hoisted from "_project_#@react-aria#i18n#@internationalized#date" <--- here it is, but it should be tree shaken...
   - Hoisted from "_project_#@react-spectrum#calendar#@internationalized#date"
   - Hoisted from "_project_#@react-spectrum#datepicker#@internationalized#date"
   - Hoisted from "_project_#@react-spectrum#labeledvalue#@internationalized#date"
   - Hoisted from "_project_#@react-stately#calendar#@internationalized#date"
   - Hoisted from "_project_#@react-stately#datepicker#@internationalized#date"
   - Hoisted from "_project_#@react-types#calendar#@internationalized#date"
   - Hoisted from "_project_#@react-types#datepicker#@internationalized#date"

**Comment by devongovett** [comment, marked as answer]
What bundler are you using? I'm not sure bundlephobia is correct here. It is a dependency of `@react-aria/i18n`, but unused by tooltip so should be tree shaken. When I bundled a simple example containing only Tooltip, it did not contain `@internationalized/date`.

**Reply by ohazda** [reply]
We use Webpack 4 and it might be also an issue but I was also surprised to see the same result on bundlephobia.

**Reply by snowystinger** [reply]
They also use webpack 4 https://github.com/pastelsky/bundlephobia/blob/bundlephobia/package.json#L91 so might be the same issue

**Reply by ohazda** [reply]
Yeah I saw it but wasn't sure if that's what they use to measure bundle stats or just for the page, and looks like this is what the [webpack fork](https://github.com/pastelsky/bundlephobia/blob/bundlephobia/package.json#L119) they use to measure it. Anyway, this is a known issue for Webpack 4 some folks reported similar behaviour under [this thread](https://github.com/webpack/webpack/issues/9337). Thank you all for looking into this 🙌🏻
